servicemesh-install/topics/install-remove.adoc

```diff
@@ -1,44 +1,81 @@
 
+[NOTE]
+====
+When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
+====
+
+[NOTE]
+====
+You can used the shortened `smcp` command in place of `servicemeshcontrolplanes`.
+====
+
 . Retrieve the name of the installed custom resource with this command:
 +
-
 ----
 oc get servicemeshcontrolplanes -n istio-system
 ----
 
+. Replace `<name_of_custom_resource>` with the output from the previous command, and run this command to remove the custom resource:
 +
-. Remove the custom resource by running the following command:
+----
+$ oc delete servicemeshcontrolplanes -n istio-system <name_of_custom_resource>
+----
+
+
+[[remove-operators]]
+== Remove the operators
+You must remove the Operators to successfully remove {ProductName}. Once you remove the {ProductName} Operator, you must remove the Jaeger Operator and the Kiali Operator.
+
+[[remove-ossm-operator]]
+=== Remove the {ProductName} Operator
 
+* Run this command to remove the {ProductName} Operator:
++
 ----
-$ oc delete -n istio-system -f <name_of_custom_resource>
+$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.12/deploy/servicemesh-operator.yaml
 ----
 
-[NOTE]
-====
-When you remove the custom resource, {ProductShortName} tells the operator to begin uninstalling everything it installed.
-====
+[[remove-jaeger-operator]]
+=== Remove the Jaeger Operator
 
+* Run the following commands to remove the Jaeger Operator:
++
+----
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/operator.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role_binding.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/role.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/service_account.yaml
+$ oc delete -n observability -f https://raw.githubusercontent.com/jaegertracing/jaeger-operator/v1.13.1/deploy/crds/jaegertracing_v1_jaeger_crd.yaml
+----
 
-[[remove-operator]]
-== Remove the operator
-Remove the operator by running the following commands:
+[[remove-kiali-operator]]
+=== Remove the Kiali Operator
 
+* Run the following command to remove the Kiali Operator:
++
 ----
-$ oc delete -n istio-operator -f https://raw.githubusercontent.com/Maistra/istio-operator/maistra-0.11/deploy/servicemesh-operator.yaml
+$ bash <(curl -L https://git.io/getLatestKialiOperator) --uninstall-mode true --operator-watch-namespace '**'
 ----
 
+
 [[remove-projects]]
 == Remove the projects
-Remove the `istio-system` and `istio-operator` projects with the following commands:
 
+. Run this command to remove the `istio-system` project:
++
 ----
 $ oc delete project istio-system
+----
+
+. Run this command to remove the `istio-operator` project:
++
+----
```

Comment:
We're we creating four namespaces/projects?  It looks like we're only deleting two of them here.

* istio-operator project (1 pod)
* istio-system project (17 pods)
* kiali-operator project (1 pod)
* observability project (1 pod)

Reply:
The kiali-operator project and observability project are covered in the removing the Kiali operator and removing the Jaeger operator sections above, but it doesn't look like the actual project gets removed in either.

Reply:
I updated the removal process.